.deployment/roles/opencast/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: stop opencast
+  service:
+    name: opencast
+    state: stopped
+  ignore_errors: true
+
+- name: remove previous deployment
+  ansible.builtin.file:
+    path: /opt/opencast-dist-allinone
+    state: absent
+
+- name: extract distribution
+  ansible.builtin.unarchive:
+    src: '{{ lookup("fileglob", "../../../build/opencast-dist-allinone-*.tar.gz") }}'
+    dest: /opt
+    owner: root
+    group: root
+
+- name: link installation from /opt/opencast
+  ansible.builtin.file:
+    src: /opt/opencast-dist-allinone
+    dest: /opt/opencast
+    owner: root
+    group: root
+    state: link
+
+- name: temporary fix for https://github.com/opencast/opencast/pull/2790
+  ansible.builtin.file:
+    path: /opt/opencast/bin/start-opencast
+    mode: '0755'
+
+- name: fix for missing configuration
+  ansible.builtin.file:
+    path: /opt/opencast/etc
+    owner: opencast
+    group: opencast
```

Comment:
Not sure how this name fits to what is actually being done here. Can you elaborate?

Reply:
The distribution we build is missing some configuration files. By allowing Opencast to write to the configuration directory, we allow them to be generated dynamically. Otherwise, Opencast would fail to start up. I've started pushing patches for this upstream, but this will fix the issue for now.

Reply:
Ah I see!